features/upgrade/storage/upgrade.feature

```diff
@@ -172,3 +172,141 @@ Feature: Storage upgrade tests
     Given the expression should be true> cluster_operator('storage').condition(type: 'Degraded')['status'] == "False"
     Given the expression should be true> cluster_operator('storage').condition(type: 'Upgradeable')['status'] == "True"
 
+  # @author chaoyang@redhat.com
+  # @case_id OCP-28630
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: Snapshot operator should be in available status after upgrade and can created pod with snapshot - prepare
+    Given the master version >= "4.5"
```

Comment:
Just a query, the case could not be run on 4.4? like upgrade from 4.4.x to 4.4.y or 4.4.x to 4.5?

Reply:
Have updated from 4.4